src/content/docs/webdev-hosting/https-ssl/ssl-https.md

```diff
@@ -0,0 +1,61 @@
+- SSL 2.0 released in 1995 as the first public available version. Provided encryption for web communications but had several security flaws.
+- SSL 3.0 released in 1996 and adressed the vulnerabilities of its predecessor. It became widely adapted and laid the 
+fondation of the HTTPS protocol.
+- in 1999, The internet Engineering Tast Force took over the standarization process for SSL, which resulted in the creation of TLS1.0(Transport Layer Security), which is essentially SLL 3.1.
+- TLS 1.1 was introduced in 2006, and adressed previous vulnerabilities,
+- TLS 1.2 released in 2008 further improving it.
+- as TLS was further developed, HTTPS became the standard for secure web communication.
+- In the early 2010s, there was a renewed emphasis on improving web security. The major browsers started encouraging the implementation of HTTPS, by displaying security warnings for non secure sites.
+- Today, HTTPS is considered the standard for securing communication on the web and websites are encouraged to implement the technonology in order to secure the privacy and integrity of user data.
+
+## HTTP
+Hypertext Transfer Protocol is the foundation of the World Wide Web. HTTP is also a set of rules which browsers use to determine how to read and transfer data over the web. When a URL contains "http", the browser knows how to follow this protocol to successfully load the webpage.
+
+## HTTPS
+HTTPS is the secure version of its predecessor HTTP. HTTP is the primary protocol used to send data between a web browser
+and a website. HTTPS does the same thing, but it encrypts the data in order to improve security.
+HTTPS uses an encryption protocol to secure communications. This protocol is called Transport Layer Security(TLS), previously known as SSL. TLS secures communications by using an asymmetric public key infrastructure. This means it uses two different keys to encrypt data sent between two parties.
+- The private key: This key is controlled by the owner of a website and is kept private. This key is stored on a web server and is used to decrypt information which is encrypted by the public key.
+- The public key: This key is not private, and is available to anyone that wants to interact with the server in a secure way.
+When a user accesses a webpage, the webpage transmits its SSL certificate, containing the public key for initiating a secure session. The client and server undergo an SSL/TLS handshake, a sequence of exchanges designed to establish a secure connection.
+
+## Benefits of encryption
+HTTPS prevents data sent from a user to a webserver, or opposite, from being openly viewed by anyone snooping on the network. Data sent over regular HTTP is broken into packets of data which can easily be sniffed by using software like wireshark. Using HTTPS is particularly important when using public wifi, because anyone connected to this wifi can view your data in plain text, and makes you particularly vulnerable to on-path attacks etc.
```

Comment:
```suggestion
HTTPS prevents data sent from a user to a webserver, or the opposite, from being openly viewed by anyone snooping on the network. Data sent over regular HTTP is broken into packets of data which can easily be sniffed by using software like Wireshark. Using HTTPS is particularly important when using public wifi, because anyone connected to this wifi can view your data in plain text, and makes you particularly vulnerable to on-path attacks etc.
```